test/integration/scheduler_perf/.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore log files
+scheduler_perf.test.*
```

Comment:
@bsalamat whatever we go with we should have this, as the `log_dir` `.` flag causes all the log files to be in the test directory and picked up by git otherwise.

I don't run these tests myself so I'm not sure if these are the actual defaults we want etc., but this implementation _should_ be slightly more robust :sweat_smile:

Reply:
switching branches will also cause these to be picked up, it's not clear to me that defaulting to `.` for the log_dir is actually a good idea...

Reply:
what is your main reason to add `log_dir=.`?

Reply:
the previous PR does https://github.com/kubernetes/kubernetes/pull/78462/files#diff-dac3f0d616ae45f98d45d6420ed1be65R884